src/Sylius/Bundle/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -88,6 +88,7 @@ sylius:
       images: Images
       on_hand: Stock actuel
       price: Prix
+      original_price: Prix original
```

Comment:
We should modify only `en` translations and then the translation process is handled on Crowdin (http://translate.sylius.org)